src/olympia/devhub/tests/test_views.py

```diff
@@ -3835,3 +3835,29 @@ def test_devhub_version_edit_page(self):
     def test_devhub_version_list_page(self):
         url = reverse('devhub.addons.versions', args=[self.addon.slug])
         self.assertNameAndNoXSS(url)
+
+
+class TestSwitchToUnlisted(UploadTest):
```

Comment:
I agree that we should clean this up at some point, as tests about changing an add-on to unlisted don't really need to be `test_views_versions.py` as they're not version specific.

Might make sense to do it when devhub gets rewritten...